pyriksdagen/utils.py

```diff
@@ -516,3 +516,29 @@ def get_context_sequences_for_protocol(protocol, context_type, target_length = 1
     output_dict = {'id' : id_list,
                    'text' : texts_with_contexts}
     return output_dict
+
+
+def pathize_protocol_id(protocol_id):
+    """
+    Turn the protocol id into a path string from riksdagen-records root. Handles 'unpadded' protocol numbers.
+    """
+
+    spl = protocol_id.split('-')
+    py = spl[1]
```

Comment:
I would write out the abbreviations for maintainability

Reply:
[8006e61](https://github.com/swerik-project/pyriksdagen/pull/18/commits/8006e612e5069ba9fd4557cc42369f6fe1eaacaa)